rust/connlib/snownet/src/node.rs

```diff
@@ -1261,9 +1204,21 @@ struct InitialConnection {
 
     created_at: Instant,
     intent_sent_at: Instant,
+
+    is_failed: bool,
 }
 
 impl InitialConnection {
+    fn handle_timeout<TId>(&mut self, id: TId, now: Instant)
+    where
+        TId: fmt::Display,
+    {
+        if now.duration_since(self.created_at) >= Duration::from_secs(20) {
```

Comment:
```suggestion
        if now.duration_since(self.created_at) >= Duration::from_secs(CXN_TIMEOUT_SECS) {
```

I know it's not duplicated, but it would be easier to find the timeout constants if they're extracted somewhere